src/main/java/frc/robot/subsystems/outtake/Outtake.java

```diff
@@ -166,7 +181,9 @@ public void setPivotBrakeMode() {
 
   public boolean isFlyWheelUpToSpeed() {
     return OrangeMath.equalToEpsilon(
-        inputs.rightRotationsPerSec, targetRPS, OuttakeConstants.outtakeToleranceRPS);
+        inputs.bottomRotationsPerSec, -bottomTargetRPS, OuttakeConstants.outtakeToleranceRPS) && 
```

Comment:
i'm going to assume the negative sign is correct here, but in the future we might just want to log absolute rps